udata/features/territories/views.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import csv
```

Comment:
You might not have to deal with encoding issues if using `unicodecsv`, already a dependency.